department-of-veterans-affairs/caseflow
# Queue | Display issue note in remand reasons screen

Resolves #5510 

### Description
Displays issue notes in Remand Reasons screen

### Acceptance Criteria 
- [ ] Issue notes are displayed in Remand Reasons screen

### Testing Plan
1. Go to `/queue`, start Draft Decision checkout flow
2. Set ≥ 1 issue disposition to Remanded, go to Select Remand Reasons screen
3. Confirm `Note: ${issue.note}` appears for each remanded issue

![screen shot 2018-05-14 at 2 31 02 pm](https://user-images.githubusercontent.com/8533004/40016172-7337a310-5783-11e8-8cc1-adefb81a5ade.png)
